src/vm/dllimport.cpp

```diff
@@ -6280,23 +6280,32 @@ INT_PTR NDirect::GetNativeLibraryExport(NATIVE_LIBRARY_HANDLE handle, LPCWSTR sy
     return address;
 }
 
+#ifndef PLATFORM_UNIX
+BOOL IsWindowsAPI(PCWSTR wszLibName)
```

Comment:
This does not return whether the name is Windows API. It returns whether the name is Windows API set. Change the name to `IsWindowsAPISet` ?